misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobHandlerModule.kt

```diff
@@ -13,45 +14,48 @@ import kotlin.reflect.KClass
  * Install this module to register a handler for an SQS queue.
  */
 class AwsSqsJobHandlerModule<T : JobHandler> private constructor(
-  private val queueName: QueueName,
+  private val queueNames: List<QueueName>,
```

Comment:
this is not backwards compatible with existing clients that install a single queue. Plus we should have a flavor of the constructor that allows a single queue, since that's the common case

Reply:
The constructor is private. There's a overload of the factory method for the single-queue case.